tools/BUILD.bazel

```diff
@@ -0,0 +1,60 @@
+# Note: this isn't actually generated at build time, but you can run this rule
+# and then copy the generated CROSSTOOL file into the tree.
```

Comment:
Plesae make hack/update-crosstool.sh and hack/verify-crosstools.sh scripts to do this

Maybe make make it bounce if bazel is not on the machine unless a flag is set (which set set for ci). 

Alternatively how about a sh_binary (to copy over the generated file to tree) and sh_test (to check that the generated file is the same as what is in tree).

That way the sh_test will only run in the bazel test //... scenario.

Reply:
I like the `sh_test` approach - it can be as easy as `diff generated_file checked-in_file`. Will work on that.